assets/js/main.js

```diff
@@ -8,11 +8,16 @@ $('body').on({
 
     if (submenu[0] && submenu[0].children.length) {
       $('.header-main').addClass('header-open');
+    } else {
+      $('.sub-menu.open').hide();
     }
   },
   mouseleave: function() {
-    if(!$(this).hasClass('submenu-item')) {
```

Comment:
in the new changes, _this_ is not used anymore, the affected element is not important anymore?

Reply:
I don't get what you mean. What new changes? Other than the ones in this PR?

Reply:
I use .header-main class that is the main class of the header. I suppose with this is more than ok.